docker-compose.yml

```diff
@@ -1,19 +1,90 @@
 services:
   data-store:
-      build:
-        context: ../funding-service-design-post-award-data-store
-      command: bash -c "flask run -p 8080 -h 0.0.0.0"
-      volumes: ['../funding-service-design-post-award-data-store:/app']
-      ports:
-        - 4001:8080
+    build:
+      context: ../funding-service-design-post-award-data-store
+    command: bash -c "flask run -p 8080 -h 0.0.0.0"
+    volumes: [ '../funding-service-design-post-award-data-store:/app' ]
+    ports:
+      - 4001:8080
+  authenticator:
+    build:
+      context: ../funding-service-design-authenticator
+    volumes: [ '../funding-service-design-authenticator:/app' ]
+    command: bash -c "python -m debugpy --listen 0.0.0.0:5678 -m flask run -p 8080 -h 0.0.0.0"
+    ports:
+      - 4004:8080
+      - 5684:5678
+    depends_on:
+      - account-store
+      - fund-store
+      - redis-data
+    environment:
+      - REDIS_INSTANCE_URI=redis://redis-data:6379
+      - ACCOUNT_STORE_API_HOST=http://account-store:8080
+      - APPLICATION_STORE_API_HOST=http://application-store:8080  # not used
+      - FUND_STORE_API_HOST=http://fund-store:8080  # required
+      - AUTHENTICATOR_HOST=http://localhost:4004
+      - NOTIFICATION_SERVICE_HOST=http://notification:8080  # not used
+      - POST_AWARD_FRONTEND_HOST=http://localhost:4002
+      - APPLICANT_FRONTEND_HOST=http://localhost:3008
+      - ASSESSMENT_FRONTEND_HOST=http://localhost:3010
+      #  Uncomment to test Azure AD locally once credentials are set in .env
+      - AZURE_AD_CLIENT_ID=${AZURE_AD_CLIENT_ID:?err}
+      - AZURE_AD_CLIENT_SECRET=${AZURE_AD_CLIENT_SECRET:?err}
+      - AZURE_AD_TENANT_ID=${AZURE_AD_TENANT_ID:?err}
```

Comment:
Do we want to comment these out so it won't break if people don't have those credentials?